feat: batch query

hredestig opened the pull request:
- faster as only touch nodes separated by max sep
- batch query by post (get repeat(string()) not supported by venom)
- return dict instead to facilitate down-stream use

this removes the old get interface as didn't see the need for it, but could just as well have kept it of course

hredestig committed:
fix: remove local volume specification

hredestig committed:
refactor: put data files in own ignored dir

hredestig committed:
feat: file to load a csv with two-cols and is rel

Load a csv file with two columns of identifier related as is-a.

hredestig committed:
feat: script to build a test database

hredestig committed:
refactor: adjust defaults for loading

- skip leading hashes for metanetx file
- use 4 threads instead of 20
- adjust env var names

hredestig committed:
feat: batch query

- faster as only touch nodes separated by max sep
- batch query by post (get repeat(string()) not supported by venom)
- return dict instead to facilitate down-stream use

Midnighter commented on build-test-db.sh:
```diff
@@ -0,0 +1,13 @@
+
+export ID_MAPPER_API=http://localhost
+export ID_MAPPER_PORT=7474
+export ID_MAPPER_PASSWORD=1
```
As per our discussion it would be better to define these environment variables in a `.env` file. Which will then be picked up by `docker-compose` as well. You can then use:

```bash
set -a
source .env
set +a
```
inside of the script if you need the values. (`set -a` exports the read in variables such that they are available for the Python scripts.)

hredestig replied:
Good idea, fixed now

sgalkina commented:
should there be `devel` branch for this service as well? this pr can be merged to master though since nothing is dependent on the service currently

hredestig committed:
chore: fix local service config

The contents of .env are used to populate docker-compose.yml. Variables
set in the shell have precedence. Note this is unrelated to the
`env_file` option which as far as I can see doesn't give an option for
shell-variable priority.

Midnighter commented on build-test-db.sh:
```diff
@@ -0,0 +1,12 @@
+set -a
+source .env
+set +a
+ID_MAPPER_API=http://localhost
```
Do you need to `export` the change or does this work as intended?

hredestig replied:
only need it in that shell so that'll work as intended

hredestig commented:
yes we should have devel branch whenever someone is actually using this service
